v2/orders/order-processing.rst

```diff
@@ -4,3 +4,39 @@ Order Processing
 Order processing is part of the order refresh process. This is run when
 on draft orders to ensure that it has up to date adjustments and that
 its order items are up to date.
+
+In Commerce Core, the order process moves throug, h the following cycle: Cart -> Validation -> Fulfillment -> Completed. It starts with the order being in the shopping cart, which is the Cart state, then, once the order is placed, it is put in the Validation state. Once we're ready to ship the goods, the order is moved to the Fulfillment state. And finally, once it leaves our store, the order is officially Completed.
```

Comment:
typo in "through" of the first sentence. Let's take the time to use lucidchart.com and create a nicer graph.